src/widget/widget.cpp

```diff
@@ -1462,13 +1462,9 @@ void Widget::onGroupNamelistChanged(int groupnumber, int peernumber, uint8_t Cha
         g = createGroup(groupnumber);
     }
 
-    QString name = Nexus::getCore()->getGroupPeerName(groupnumber, peernumber);
     TOX_CHAT_CHANGE change = static_cast<TOX_CHAT_CHANGE>(Change);
     if (change == TOX_CHAT_CHANGE_PEER_ADD)
     {
-        if (name.isEmpty())
-            name = tr("<Unknown>", "Placeholder when we don't know someone's name in a group chat");
-
```

Comment:
In this function `name` variable was used even before my changing only in 

    else if (change == TOX_CHAT_CHANGE_PEER_NAME) // core overwrites old name before telling us it changed...
    {
        g->updatePeer(peernumber, name);
    }

block. So checking for emptiness in the `f (change == TOX_CHAT_CHANGE_PEER_ADD)...` is useless and never worked before.